### PR TITLE
Bug 1762695: kubernetes 1.14.8 and pin yaml for billion laughs

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

**Comment by mfojtik** [comment]
cc @deads2k @sttts

File: glide.yaml [hunk 1]
```diff
@@ -74,13 +74,13 @@ import:
 
 # openshift second
 - package: github.com/openshift/api
-  version: master
+  version: release-4.2
```
Comment (mfojtik) [inline glide.yaml]: @deads2k this also pins release-4.2 to release-4.2 ;-)

**Comment by mfojtik** [comment]
FYI: the `release-4.2` branch in apiserver-library-go was not promoted/created, I did that manually and based it on glide.lock from this repo. https://github.com/openshift/release/pull/5483 should fix this for `release-4.3`.

File: glide.yaml (hunk 1 above)
Comment (sttts) [reply glide.yaml]: very helpful

File: glide.yaml [hunk 2]
```diff
@@ -74,13 +74,13 @@ import:
...
 - package: github.com/openshift/library-go
-  version: master
+  version: release-4.2
 - package: github.com/openshift/apiserver-library-go
-  version: master
+  version: release-4.2
 - package: github.com/openshift/source-to-image
   version: master
```
[the remark this answers is not in the document]
Comment (soltysh) [reply glide.yaml]: nope, I've already asked, they have disconnected releases

**Pull request closed without merging**